docs/intro.md

```diff
@@ -16,12 +16,13 @@ The Subspace Network is an ambitious layer zero protocol which is the first scal
 ---
 - [What is the Subspace Network](https://subspace.network/technology)
 - [Frequently Asked Questions](https://subspace.network/faq)
+- [Subspace Forum - Space for deeper discussions](https://forum.subspace.network/)
```

Comment:
I'm afraid that "Space for deeper discussions" might be a bit unclear, wdyt about 
"Subspace Forum - In-Depth Discissions"?